type.proto

```diff
@@ -271,7 +271,7 @@ message StopPoint {
     repeated PhysicalMode physical_modes = 18;
     optional FareZone fare_zone = 19;   
     repeated EquipmentDetails equipment_details = 20;
-
+    repeated Line lines = 22;
```

Comment:
There is already a number `22` for   `repeated string impact_uris = 22;`
According to protobuff doc ` each field in the message definition has a unique number. `
```suggestion
    repeated Line lines = 23;
```

Reply:
Thanks :+1: Done